### PR TITLE
done wishlist page

### PR DESCRIPTION
**Pull request by srishtigautam21**
Done wishlist page 
added logout button on navbar
Made slider input for  ratings in sidebar

### CODE REVIEW

File: Pages/ProductPage/product.css [hunk 1]
```diff
@@ -67,3 +77,19 @@
 .margin-top{
     margin-top: 1rem;
 }
+.rating-margin{
+    margin:5px 0;
+}
+.slider-range{
+    display: flex;
+    /* justify-content:space-between; */
```
Comment (archit7448) [inline Pages/ProductPage/product.css]: remove commented codes

**Review by archit7448 — APPROVED** [review]
good to merge

File: Pages/ProductPage/product.css (hunk 1 above)
Comment (zoef-yt) [inline Pages/ProductPage/product.css]: remove commented code.

File: Pages/ProductPage/product.css [hunk 2]
```diff
@@ -67,3 +77,19 @@
...
+}
+.slider{
+  /* -webkit-appearance: none; */
```
Comment (zoef-yt) [inline Pages/ProductPage/product.css]: remove commented code.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by zoef-yt — APPROVED** [review]
Good to merge.

**Review by itsmebipul00 — APPROVED** [review]

**Pull request merged**